samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Shared/AdapterSpecificHeaders.cs

```diff
@@ -0,0 +1,4 @@
+﻿public class AdapterSpecificHeaders
```

Comment:
What is this for? I couldn't see any place where it's used directly. Is this some sort of hidden requirement for using the adapter? If so, is the documented anywhere?